tutorials/install-postgresql/index.mdx

```diff
@@ -7,8 +7,8 @@ content:
   paragraph: This page shows how to install PostgreSQL on your server
 tags: compute networking database sql postgresql server
 categories:
-  - storage
-  - compute
+  - postgresql-and-mysql
```

Comment:
Same question here (and for others) - if they have the category of a particular product (postgresql-and-mysql, instances) should they also have the overarching category for that product (managed-databases, compute) ? 

Looking at the tutorials filter, I guess it doesn't make sense because there is no clickable filter for the overarching category, only the product. But then some tutorials below do indeed have "compute", "managed-databases" as categories :exploding_head: 

Maybe we can clarify this with the team?